packages/editor/src/client/routes/articleList.tsx

```diff
@@ -337,8 +351,32 @@ export function ArticleList() {
                   break
 
                 case ConfirmAction.Duplicate:
-                  await duplicateArticle({
```

Comment:
I guess the await is important here?

Reply:
Not needed anymore because it's a Promise with a then block